### PR TITLE
New quotes from movies like Snowden, Passengers, Mr. Robot, The Martian, Final Destination and Joker.

### PR DESCRIPTION
**Pull request by mehtadhiraj**
Care is taken that none of the quote is repeated, though the json contain many redunducy.

### CODE REVIEW

File: data/data.json [hunk 1]
```diff
@@ -5839,7 +5839,73 @@
...
+     "quote": "Secrecy is security and security is victory.",
+     "movie": "Snowden",
+     "type": "movie",
+     "year": 2016
+   },
+   {
+     "quote":"There are No Accidents. No Coincidences. No Escapes. YOU CAN'T CHEAT DEATH.",
+     "movie": "Final Destination",
+     "type": "Series",
+     "year": 2000
```
Comment (NikhilNamal17) [inline data/data.json]: Rename to type: tv

Comment (mehtadhiraj) [reply data/data.json]: Ok will do the required changes

**Comment by stevesioli** [comment]
@NikhilNamal17 the branch for this PR was not merged properly, and resulted in the duplication of the entire original data.

**Comment by NikhilNamal17** [comment]
@stevesioli I fixed it.

**Pull request merged**